src/main/java/Duke.java

```diff
@@ -1,10 +1,154 @@
+import java.util.*;
 public class Duke {
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
+        showWelcomeMessage(logo);
+        Scanner in = new Scanner(System.in);
+        String input = getAndProcessInput(in);
+        Boolean inputNotValid = Boolean.TRUE;
```

Comment:
You might want to consider naming your Boolean variables such that it sounds like a Boolean e.g. isInputValid